sherpa/models/basic.py

```diff
@@ -1553,6 +1556,15 @@ def __init__(self, name='sigmagauss2d'):
                                   self.ypos, self.theta, self.ampl))
         self.cache = 0
 
+    def guess(self, dep, *args, **kwargs):
+        xpos, ypos = guess_position(dep, *args)
+        norm = guess_amplitude2d(dep, *args)
```

Comment:
This isn't used. I think we would want the ampl to be guessed too (it is in Gauss2D).

Reply:
Added the guessed ampl:
```
(sherpa) [dtn@devel12 sherpa]$ git diff models/basic.py
diff --git a/sherpa/models/basic.py b/sherpa/models/basic.py
index 2e0784e1..799fed36 100644
--- a/sherpa/models/basic.py
+++ b/sherpa/models/basic.py
@@ -1564,6 +1564,7 @@ class SigmaGauss2D(Gauss2D):
         param_apply_limits(ypos, self.ypos, **kwargs)
         param_apply_limits(fwhm, self.sigma_a, **kwargs)
         param_apply_limits(fwhm, self.sigma_b, **kwargs)
+        param_apply_limits(norm, self.ampl, **kwargs)
 
     def calc(self, *args, **kwargs):
         kwargs['integrate'] = bool_cast(self.integrate)
```